backend/src/modules/routes/accountRoutes.ts

```diff
@@ -0,0 +1,11 @@
+import { SubRoutes } from 'enums/routesEnum';
+import { Router } from 'express';
+
+import { loginController, refreshController } from 'modules/controllers/authController';
```

Comment:
Everything is messed up again. Try to separate all the imports by 3 groups:

- lib imports
- absolute imports
- relative imports